### PR TITLE
2nd try

### PR DESCRIPTION
**Pull request by dariotrombello**
I categorized the commits now and applied the changes onto the latest Exynos7580 a3xeltexx device tree.

### CODE REVIEW

File: overlay/frameworks/base/core/res/res/values/config.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 /*
-** Copyright 2013, The CyanogenMod Project
```
Comment (Stricted) [inline overlay/frameworks/base/core/res/res/values/config.xml:4]: dont remove copyrights

**Review by Stricted — CHANGES_REQUESTED** [review]

File: proprietary-files.txt [hunk 2]
```diff
@@ -25,7 +25,7 @@ vendor/firmware/bcm43438A1_V0051.0217.hcd
 # Camera
 ###################
 
-lib/hw/camera.universal7580.so
+lib/hw/camera.vendor.universal7580.so
```
Comment (danwood76) [inline proprietary-files.txt]: This is not correct. The proprietary-files.txt is what is dumped from a vendor image (or current lineageos image).

As we currently rename this blob so we can use my camera wrapper this is probably more suitable:
lib/hw/camera.universal7580.so:lib/hw/camera.vendor.universal7580.so

**Review by danwood76 — CHANGES_REQUESTED** [review]

**Comment by dariotrombello** [comment]
Satisfied now? 😆

**Comment by Stricted** [comment]
i guess you are not familiar with `git rebase` and `git commit --amend`?

**Comment by dariotrombello** [comment]
I know git commit --amend. I think you can change commits with that but i never used it.

File: proprietary-files.txt (hunk 2 above)
Comment (dariotrombello) [reply proprietary-files.txt]: Thanks for the information

**Comment by danwood76** [comment]
I have picked your changes and fixed the issues.

**Pull request closed without merging**